--- libffado-2.3.0/src/libutil/TimestampedBuffer.cpp	(revision 2628)
+++ libffado-2.3.0/src/libutil/TimestampedBuffer.cpp	(revision 2629)
@@ -32,6 +32,7 @@
 
 #include <cstdlib>
 #include <cstring>
+#include <math.h>
 
 #define DLL_PI        (3.141592653589793238)
 #define DLL_SQRT2     (1.414213562373095049)
@@ -179,8 +180,6 @@
     m_wrap_at=w;
     return true;
 }
-#include <math.h>
-
 /**
  * \brief return the effective rate
  *
