--- a/support/mixer-qt4/ffado/mixer/saffire_dice.py	2017-06-14 22:01:08.176757989 +0900
+++ b/support/mixer-qt4/ffado/mixer/saffire_dice.py	2017-06-14 22:00:48.534783458 +0900
@@ -69,7 +69,7 @@
 
         widget = QtGui.QWidget()
 
-        ModelName = self.configrom.getModelName()
+        ModelName = self.configrom.getModelName().strip()
         if  ModelName == "SAFFIRE_PRO_14":
             uicLoad("ffado/mixer/Saffire_Pro14_monitoring.ui", widget)
         elif ModelName == "SAFFIRE_PRO_24" or self.configrom.getModelName() == "SAFFIRE_PRO_24DSP":
