From 99774ac4c89e002f1225862699d62bab58ee4cb2 Mon Sep 17 00:00:00 2001
From: Simon van der Veldt <simon.vanderveldt@gmail.com>
Date: Mon, 11 Feb 2019 21:03:09 +0100
Subject: [PATCH] Makefile: Don't compress man pages

---
 Makefile.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile.in b/Makefile.in
index 7ab4e5d..256327f 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -163,7 +163,6 @@ install_jack:	jack $(name).1
 	@$(MAKE) INSTALL_ROOT=$(DESTDIR) -f $(name_jack).mak install
 	@install -d -v -m 0755 $(DESTDIR)$(mandir)/man1
 	@install -v -m 0644 $(name)*.1 $(DESTDIR)$(mandir)/man1
-	@gzip -vf $(DESTDIR)$(mandir)/man1/$(name)*.1
 
 
 uninstall_core:	core
-- 
2.19.2

