From a2a17cd39220e2013d3402b06a53f5a46492aa00 Mon Sep 17 00:00:00 2001
From: Nedko Arnaudov <nedko@arnaudov.name>
Date: Sun, 7 Oct 2012 14:06:04 +0300
Subject: [PATCH] add includes for getrlimit

---
 daemon/loader.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/daemon/loader.c b/daemon/loader.c
index b9fe6f1c..37fb94fa 100644
--- a/daemon/loader.c
+++ b/daemon/loader.c
@@ -34,6 +34,8 @@
 #include <fcntl.h>
 #include <pty.h>                /* forkpty() */
 #include <sys/wait.h>
+#include <sys/time.h>
+#include <sys/resource.h>
 
 #include "loader.h"
 #include "../proxies/conf_proxy.h"
-- 
2.13.0

