diff --git a/wscript b/wscript
index a7594716..5f96cd30 100644
--- a/wscript
+++ b/wscript
@@ -46,6 +46,7 @@ def options(opt):
     opt.load('boost')
     opt.load('python')
     opt.add_option('--enable-pkg-config-dbus-service-dir', action='store_true', default=False, help='force D-Bus service install dir to be one returned by pkg-config')
+    opt.add_option('--enable-gladish', action='store_true', default=False, help='Build gladish')
     opt.add_option('--enable-liblash', action='store_true', default=False, help='Build LASH compatibility library')
     opt.add_option('--enable-pylash', action='store_true', default=False, help='Build python bindings for LASH compatibility library')
     opt.add_option('--debug', action='store_true', default=False, dest='debug', help="Build debuggable binaries")
@@ -205,8 +206,7 @@ def configure(conf):
             mandatory = False,
             errmsg="not found, see http://boost.org/")

-    conf.env['BUILD_GLADISH'] = build_gui
-
+    conf.env['BUILD_GLADISH'] = Options.options.enable_gladish
     conf.env['BUILD_LIBLASH'] = Options.options.enable_liblash
     conf.env['BUILD_PYLASH'] =  Options.options.enable_pylash
     if conf.env['BUILD_PYLASH'] and not conf.env['BUILD_LIBLASH']:
