diff -Naur xjadeo-0.6.4.orig/configure.ac xjadeo-0.6.4/configure.ac
--- xjadeo-0.6.4.orig/configure.ac	2013-03-24 08:28:16.733866229 +0100
+++ xjadeo-0.6.4/configure.ac	2013-03-24 08:30:50.578861156 +0100
--- configure.ac	2015-11-25 06:29:32.000000000 +0500
+++ configure.ac.2	2016-04-13 00:43:38.108449695 +0500
@@ -280,10 +280,6 @@
 	fi
 	if test "x$FOUND_H_PORTMIDI$FOUND_L_PORTMIDI" = "xyesyes"; then
 		MIDI_LIBS="-lportmidi"
-dnl TODO check if porttime is needed for threading
-		if test "$PM_OS" != "mac"; then
-		  MIDI_LIBS="${MIDI_LIBS} -lporttime"
-	  fi
 	fi
 
 	if test "x$JACKMIDI" != "xno"; then
