From 26e14f1c7fe9b1ba0f641c9e43526ecf3235c1f0 Mon Sep 17 00:00:00 2001
From: Simon van der Veldt <simon.vanderveldt@gmail.com>
Date: Sun, 1 Oct 2017 12:21:48 +0200
Subject: [PATCH] Don't run ldconfig

---
 wscript | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/wscript b/wscript
index 2b4e27d..4757b21 100644
--- a/wscript
+++ b/wscript
@@ -47,19 +47,6 @@ def makelib(bld,*k,**kw):
     kw['name'] = kw['target'] + '_shared'
     bld.stlib(*k,**kw)
 
-# from autowaf
-def run_ldconfig(ctx):
-    if (ctx.cmd == 'install'
-        and not ctx.env['RAN_LDCONFIG']
-        and ctx.env['LIBDIR']
-        and not os.environ.get('DESTDIR')):
-        try:
-            Logs.info("Waf: Running `/sbin/ldconfig %s'" % ctx.env['LIBDIR'])
-            subprocess.call(['/sbin/ldconfig', ctx.env['LIBDIR']])
-            ctx.env['RAN_LDCONFIG'] = True
-        except:
-            pass
-    
 def options(opt):
     opt.load('compiler_c')
     opt.load('compiler_cxx')
@@ -552,7 +539,5 @@ src/Fl_Gl_Window.cxx
     bld.install_files( bld.env.INCLUDEDIR + '/ntk/FL', start_dir.ant_glob('*.H *.h'),
                        cwd=start_dir, relative_trick=True)
 
-    bld.add_post_fun( run_ldconfig )
-
     #  bld.install_files( string.join( [ '${DATADIR}/doc', APPNAME ], '/' ), bld.path.ant_glob( 'doc/*.html doc/*.png' ) )
     
-- 
2.13.6

