Index: b/ntk.pc.in
===================================================================
--- a/ntk.pc.in
+++ b/ntk.pc.in
@@ -1,6 +1,6 @@
 prefix=@PREFIX@
 exec_prefix=${prefix}
-libdir=${exec_prefix}/lib
+libdir=@LIBDIR@
 includedir=${prefix}/include
 
 Name: ntk
Index: b/ntk_gl.pc.in
===================================================================
--- a/ntk_gl.pc.in
+++ b/ntk_gl.pc.in
@@ -1,6 +1,6 @@
 prefix=@PREFIX@
 exec_prefix=${prefix}
-libdir=${exec_prefix}/lib
+libdir=@LIBDIR@
 includedir=${prefix}/include
 
 Name: ntk_images
Index: b/ntk_images.pc.in
===================================================================
--- a/ntk_images.pc.in
+++ b/ntk_images.pc.in
@@ -1,6 +1,6 @@
 prefix=@PREFIX@
 exec_prefix=${prefix}
-libdir=${exec_prefix}/lib
+libdir=@LIBDIR@
 includedir=${prefix}/include
 
 Name: ntk_images
Index: b/wscript
===================================================================
--- a/wscript
+++ b/wscript
@@ -456,7 +456,8 @@ src/Fl_Gl_Window.cxx
          install_path = '${LIBDIR}/pkgconfig',
          CFLAGS = ' '.join( CFLAGS ),
          VERSION = VERSION,
-         PREFIX = bld.env.PREFIX )
+         PREFIX = bld.env.PREFIX,
+         LIBDIR = bld.env.LIBDIR )
 
     bld( features = 'subst',
          source = 'ntk_images.pc.in',
@@ -465,7 +466,8 @@ src/Fl_Gl_Window.cxx
          install_path = '${LIBDIR}/pkgconfig',
          CFLAGS = ' '.join( CFLAGS ),
          VERSION = VERSION,
-         PREFIX = bld.env.PREFIX )
+         PREFIX = bld.env.PREFIX,
+         LIBDIR = bld.env.LIBDIR )
 
     bld( features = 'subst',
          source = 'ntk_gl.pc.in',
@@ -474,7 +476,8 @@ src/Fl_Gl_Window.cxx
          install_path = '${LIBDIR}/pkgconfig',
          CFLAGS = ' '.join( CFLAGS ),
          VERSION = VERSION,
-         PREFIX = bld.env.PREFIX )
+         PREFIX = bld.env.PREFIX,
+         LIBDIR = bld.env.LIBDIR )
 
     bld( features = 'subst',
          source = 'ntk-uninstalled.pc.in',
