From 62b9004266ca9ea0ac49d7f6bdfc1af4e27a4a61 Mon Sep 17 00:00:00 2001
From: Simon van der Veldt <simon.vanderveldt@gmail.com>
Date: Sun, 1 Oct 2017 12:46:32 +0200
Subject: [PATCH] Remove default C(XX)FLAGS optimizations

---
 wscript | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/wscript b/wscript
index 4757b21..192d02f 100644
--- a/wscript
+++ b/wscript
@@ -195,14 +195,6 @@ def configure(conf):
     # FIXME: use a test for this
     conf.define( 'WORDS_BIGENDIAN', 0 )
 
-    optimization_flags = [
-        "-O3",
-        "-fomit-frame-pointer",
-        "-ffast-math",
-        #            "-fstrength-reduce",
-        "-pipe"
-        ]
-
     debug_flags = [
         '-g',
         '-O0' ]
@@ -219,8 +211,6 @@ def configure(conf):
         conf.env.append_value('CXXFLAGS', debug_flags )
     else:
         print('Building for performance')
-        conf.env.append_value('CFLAGS', optimization_flags )
-        conf.env.append_value('CXXFLAGS', optimization_flags )
         conf.define( 'NDEBUG', 1 )
 
     conf.env.append_value('CFLAGS', CFLAGS )
-- 
2.13.6

