diff --git a/usr/lib/praxis-live/etc/praxis.conf b/usr/lib/praxis-live/etc/praxis.conf
index ed69d4f..db15191 100644
--- a/usr/lib/praxis-live/etc/praxis.conf
+++ b/usr/lib/praxis-live/etc/praxis.conf
@@ -8,7 +8,7 @@ default_options="--nosplash --nogui --branding praxis -J-Dpraxis.version=3.1.0 -
 # for development purposes you may wish to append: -J-Dnetbeans.logger.console=true -J-ea
 
 # default location of JDK/JRE, can be overridden by using --jdkhome <dir> switch
-#jdkhome="/path/to/jdk"
+jdkhome="${JAVA_HOME}"
 
 # clusters' paths separated by path.separator (semicolon on Windows, colon on Unices)
 #extra_clusters=
diff --git a/usr/lib/praxis-live/etc/praxis_live.conf b/usr/lib/praxis-live/etc/praxis_live.conf
index 1d94484..34d0b01 100644
--- a/usr/lib/praxis-live/etc/praxis_live.conf
+++ b/usr/lib/praxis-live/etc/praxis_live.conf
@@ -8,7 +8,7 @@ default_options="--branding praxis_live -J-Dpraxis.version=3.1.0 -J-Dsun.java2d.
 # for development purposes you may wish to append: -J-Dnetbeans.logger.console=true -J-ea
 
 # default location of JDK/JRE, can be overridden by using --jdkhome <dir> switch
-#jdkhome="/path/to/jdk"
+jdkhome="${JAVA_HOME}"
 
 # clusters' paths separated by path.separator (semicolon on Windows, colon on Unices)
 #extra_clusters=
