From ff938221d03e4068d1fbcc204799a9c588bbacc2 Mon Sep 17 00:00:00 2001
From: Simon van der Veldt <simon.vanderveldt@gmail.com>
Date: Wed, 15 Nov 2017 22:09:22 +0100
Subject: [PATCH] Fix configure.ac: use rtmidi as default backend

Fixes #116
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index d607feb1..c87a0e8c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -522,7 +522,7 @@ dnl Eventually we want to make ALSA optional for the user/developer.
 AC_ARG_ENABLE(rtmidi,
     [AS_HELP_STRING(--enable-rtmidi, [Enable rtmidi build])],
     [rtmidi=$enableval],
-    [rtmidi=no])
+    [rtmidi=yes])
 
 if test "$rtmidi" != "no"; then
     build_rtmidi="yes"
-- 
2.13.6

