From f27deec4f439dc1b188e1599274fdb8a29a65b7e Mon Sep 17 00:00:00 2001
From: Simon van der Veldt <simon.vanderveldt@gmail.com>
Date: Mon, 21 Feb 2022 21:38:29 +0100
Subject: [PATCH] wscript: Fix duplicate manifest install

---
 wscript | 13 +++++--------
 1 file changed, 5 insertions(+), 8 deletions(-)

diff --git a/wscript b/wscript
index 26781ad..cd2fe8b 100644
--- a/wscript
+++ b/wscript
@@ -103,14 +103,11 @@ def build(bld):
         tgt = task.outputs[0].abspath()
         return shutil.copy(src, tgt)

-    for i in bld.path.ant_glob('deteriorate.lv2/*.ttl'):
-        bld(features     = 'subst',
-            is_copy      = True,
-            source       = i,
-            target       = 'deteriorate.lv2/%s' % i.name,
-            install_path = '${LV2DIR}/deteriorate.lv2')
-
-
+    bld(features     = 'subst',
+        is_copy      = True,
+        source       = 'deteriorate.lv2/manifest.ttl',
+        target       = 'deteriorate.lv2/manifest.ttl',
+        install_path = '${LV2DIR}/deteriorate.lv2')

     plugins = '''
     downsampler_mono
--
2.34.1
