From 313109cb1c41325c84f6878b8287de8d61a353d6 Mon Sep 17 00:00:00 2001
From: Simon van der Veldt <simon.vanderveldt@gmail.com>
Date: Thu, 13 Feb 2020 22:00:00 +0100
Subject: [PATCH] Fix libboost_python<xy> detection

---
 setup.py | 13 ++++++-------
 1 file changed, 6 insertions(+), 7 deletions(-)

diff --git a/setup.py b/setup.py
index 5093a0a..7869beb 100755
--- a/setup.py
+++ b/setup.py
@@ -47,12 +47,9 @@ def pkgconfig(pkg):

 def boost_lib_name(lib):
     for libdir in ('/usr/lib', '/usr/local/lib', '/usr/lib64', '/usr/local/lib64'):
-        for suffix in ('', '-mt'):
-            libname = 'lib%s%s.so' % (lib, suffix)
-            if os.path.isfile(os.path.join(libdir, libname)):
-                return lib + suffix
-    return lib + '-mt'
-
+        libname = 'lib%s.so' % (lib)
+        if os.path.isfile(os.path.join(libdir, libname)):
+            return lib

 sources = [
     'src/engine.cc',
@@ -71,7 +68,9 @@ include_dirs.append('src')

 pkgconfig('glib-2.0')

-libraries.append(boost_lib_name('boost_python'))
+python_version = '%d%d' % sys.version_info[:2]
+
+libraries.append(boost_lib_name('boost_python' + python_version))
 libraries.append(boost_lib_name('boost_thread'))

 # uncomment and adapt these to build using a custom install of boost.
--
2.24.1
