From 1ae960dc5fade7122cbe34f072fb326a6f07bea1 Mon Sep 17 00:00:00 2001
From: Simon van der Veldt <simon.vanderveldt@gmail.com>
Date: Fri, 28 Apr 2017 12:46:09 +0200
Subject: [PATCH] Allow configuring which plugin types to build

---
 Makefile                | 4 ++++
 plugins/Nekobi/Makefile | 6 ++++++
 2 files changed, 10 insertions(+)

diff --git a/Makefile b/Makefile
index a548af5..da90442 100644
--- a/Makefile
+++ b/Makefile
@@ -19,8 +19,12 @@ plugins: dgl
 	$(MAKE) all -C plugins/Nekobi
 
 ifneq ($(CROSS_COMPILING),true)
+ifeq ($(BUILD_LV2),true)
 gen: plugins dpf/utils/lv2_ttl_generator
 	@$(CURDIR)/dpf/utils/generate-ttl.sh
+else
+gen:
+endif
 ifeq ($(MACOS),true)
 	@$(CURDIR)/dpf/utils/generate-vst-bundles.sh
 endif
diff --git a/plugins/Nekobi/Makefile b/plugins/Nekobi/Makefile
index e9b6de7..c0d6eff 100644
--- a/plugins/Nekobi/Makefile
+++ b/plugins/Nekobi/Makefile
@@ -36,20 +36,26 @@ ifeq ($(HAVE_JACK),true)
 TARGETS += jack
 endif
 
+ifeq ($(BUILD_DSSI),true)
 TARGETS += dssi_dsp
 ifeq ($(HAVE_DGL),true)
 ifeq ($(HAVE_LIBLO),true)
 TARGETS += dssi_ui
 endif
 endif
+endif
 
+ifeq ($(BUILD_LV2),true)
 ifeq ($(HAVE_DGL),true)
 TARGETS += lv2_sep
 else
 TARGETS += lv2_dsp
 endif
+endif
 
+ifeq ($(BUILD_VST),true)
 TARGETS += vst
+endif
 
 all: $(TARGETS)
 
-- 
2.26.2

