From c9996ea5d1753ed2514d99d5ca0d6440a0e27436 Mon Sep 17 00:00:00 2001
From: Simon van der Veldt <simon.vanderveldt@gmail.com>
Date: Sun, 2 Feb 2020 12:51:55 +0100
Subject: [PATCH] Makefile: Don't compress man pages

---
 Makefile.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile.in b/Makefile.in
index 3155951..1e747c1 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -159,7 +159,6 @@ install_jack:	jack
 	@$(MAKE) INSTALL_ROOT=$(DESTDIR) -f $(name_jack).mak install
 	@install -d -v -m 0755 $(DESTDIR)$(mandir)/man1
 	@install -v -m 0644 $(name)*.1 $(DESTDIR)$(mandir)/man1
-	@gzip -nvf $(DESTDIR)$(mandir)/man1/$(name)*.1
 
 
 uninstall_core:	core
-- 
2.24.1

